doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -48,35 +48,36 @@ Perform these steps to link PyWorkbench with a local session:
 - Use the given port number to link PyWorkbench with the server.
 
 ```{python}
-from ansys.workbench.core import launch_workbench
-wb = launch_workbench()
+from ansys.workbench.core import connect_workbench
+workbench = connect_workbench(port=port)
 ```
 
 ## Connect to a remote Workbench server
 
 Execute the steps below to link PyWorkbench with a remote session:
```

Comment:
```suggestion
Execute the following steps to link PyWorkbench with a remote session:
```
Google dev doc style guide says not to use placement locators like above and below but rather use preceding and following.